### PR TITLE
Base Responses to all Controllers

### PR DESCRIPTION
**Pull request by joecombopiano**
- Removed Gson dependency from RestApiHelper and controllers. 
- Added methods to return data 'AsResponse' in addition to the methods available for returning the ItemSense specific POJO's
- Updated pom.xml for json serialization issue
- Rev'd maven package (haven't pushed yet)
- Cleaned up unused dependencies

### CODE REVIEW

**Comment by joecombopiano** [comment]
@ruhkopf does this satisfy your requirement for base Responses?

**Comment by joecombopiano** [comment]
@4141done fyi on this as well

File: src/main/java/com/impinj/itemsense/client/coordinator/facility/FacilityController.java [hunk 1]
```diff
@@ -1,39 +1,55 @@
...
         return this.restApiHelper.get(facilityName, "/configuration/v1/facilities/show", target);
     }
-    public ArrayList<Facility> getAllFacilities(){
-        return this.restApiHelper.getMultiple(null, "/configuration/v1/facilities/show", target, gson);
+
+    public Response getAllFacilitiesAsResponse() {
+        return this.restApiHelper.get("/configuration/v1/facilities/show", target);
+    }
+
+    public ArrayList<Facility> getAllFacilities() {
```
Comment (bholz) [inline src/main/java/com/impinj/itemsense/client/coordinator/facility/FacilityController.java]: Return a List<Facility> not ArrayList

File: src/main/java/com/impinj/itemsense/client/coordinator/job/JobController.java [hunk 2]
```diff
@@ -1,44 +1,56 @@
...
         this.target = target;
         this.restApiHelperPoster = new RestApiHelper<Job>(Job.class);
         this.restApiHelperGetter = new RestApiHelper<JobResponse>(JobResponse.class);
     }
 
+    public Response getJobsAsResponse() {
+        return this.restApiHelperGetter.get( "/control/v1/jobs/show", target);
+    }
+
     public ArrayList<JobResponse> getJobs() {
```
Comment (bholz) [inline src/main/java/com/impinj/itemsense/client/coordinator/job/JobController.java]: Return List

File: src/main/java/com/impinj/itemsense/client/coordinator/readerconfiguration/ReaderConfigurationController.java [hunk 3]
```diff
@@ -1,45 +1,59 @@
...
     public Response deleteReaderConfiguration(String readerConfigurationName) {
         return this.restApiHelper.delete(readerConfigurationName, "/configuration/v1/readerConfigurations/destroy", target);
     }
 
     public ReaderConfiguration getReaderConfiguration(String readerConfigurationName) {
-        return this.restApiHelper.get(readerConfigurationName, "/configuration/v1/readerConfigurations/show", target);
+        return getReaderConfigurationAsResponse(readerConfigurationName).readEntity(ReaderConfiguration.class);
     }
 
     public ArrayList<ReaderConfiguration> getReaderConfigurations() {
```
Comment (bholz) [inline src/main/java/com/impinj/itemsense/client/coordinator/readerconfiguration/ReaderConfigurationController.java]: return List

File: src/main/java/com/impinj/itemsense/client/coordinator/currentZoneMap/CurrentZoneMapController.java [hunk 4]
```diff
@@ -35,11 +35,20 @@ public Response clearCurrentZoneMap(String facility) {
         return this.restApiHelper.delete(facility, "/configuration/v1/currentZoneMap/clear", target);
     }
 
+    public Response getCurrentZoneMapAsResponse(String facility){
+        if(facility != null && !facility.isEmpty()){
```
Comment (bholz) [inline src/main/java/com/impinj/itemsense/client/coordinator/currentZoneMap/CurrentZoneMapController.java]: Could use apache StringUtils so we don't need two checks for empty/null strings

**Pull request merged**